client/css/style.css

```diff
@@ -1961,6 +1961,14 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 	}
 }
 
+@media (min-width: 1610px) {
+	#windows .header .topic,
+	.messages .msg,
+	.sidebar {
+		font-size: 14px;
+	}
+}
+
```

Comment:
I'm not sure what you are referring to, the fact that these have different font sizes on a different sized screen? Or what specifically is set to a bigger size? Or what the bigger size is?